packages/strapi-plugin-documentation/services/Documentation.js

```diff
@@ -1284,12 +1286,59 @@ module.exports = {
     if (verb === 'get' && controllerMethod === 'find') {
       // parametersOptions corresponds to this section
       // of the documentation https://strapi.io/documentation/guides/filters.html
+
+      const apiElement = this.extractAPIElement(endPoint);
+      if (this.hasAPIElement(apiElement)) {
+        return this.combineMembersAndParameters(apiElement, params);
+      }
       return [...params, ...parametersOptions];
```

Comment:
OK, the upcoming commit will not return `...parametersOptions` in case the api is not matching a model. The resulting code will look like this:

```javascript
      const apiName = this.extractAPIName(routeObject);
      if (this.hasAPIName(apiName)) {
        return this.combineMembersAndParameters(apiName, params);
      }
      return [...params];
```